php/chubbyphp/config.yaml

```diff
@@ -0,0 +1,4 @@
+framework:
+  website: https://github.com/chubbyphp/chubbyphp-framework
```

Comment:
could you use `github: chubbyphp/chubbyphp-framework` ?

Reply:
done